reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -472,10 +472,7 @@
 	 * @return a new {@link Flux} concatenating all inner sources sequences
 	 */
 	public static <T> Flux<T> concat(Publisher<? extends Publisher<? extends T>> sources, int prefetch) {
-		return onAssembly(new FluxConcatMap<>(from(sources),
-				identityFunction(),
-				Queues.get(prefetch), prefetch,
-				FluxConcatMap.ErrorMode.IMMEDIATE));
+		return wrap(sources).concatMap(identityFunction(), prefetch);
```

Comment:
👍, please just mention this change in the body of the commit message

Reply:
Isn't it just another way of writing the same and completely "internal"? Or did I change the behaviour? 😱

Reply:
well, the `wrap` won't apply assembly hooks, but the `concatMap` will, so I think this is ok. it's just that the direct application of assembly hooks would have pointed tracebacks maybe a tiny bit more directly to `concat` (rather than having `concatMap` in the traceback), but I think that is ok.

Reply:
Oh, you're right! Actually, let's use `from` as it was before to avoid any side effects from this change. I will submit a separate PR that will use `wrap` where applicable and evaluate the impact without the rest of the concatMap changes, ok?

Reply:
sounds good to me